systemtest/src/test/java/io/strimzi/systemtest/ConnectS2IST.java

```diff
@@ -332,6 +335,91 @@ void testJvmAndResources() {
         KafkaConnectS2IResource.deleteKafkaConnectS2IWithoutWait(kafkaConnectS2i);
     }
 
+    @Test
+    void testKafkaConnectorWithConnectS2IAndConnectWithSameName() {
+        String topicName = "test-topic-" + new Random().nextInt(Integer.MAX_VALUE);
+        String connectClusterName = "connect-cluster";
+        String connectS2IClusterName = "connect-s2i-cluster";
+
+        KafkaResource.kafkaEphemeral(CLUSTER_NAME, 3).done();
+        // Crate connect cluster with default connect image
+        KafkaConnectS2IResource.kafkaConnectS2I(CLUSTER_NAME, CLUSTER_NAME, 1)
+            .editMetadata()
+                .addToLabels("type", "kafka-connect-s2i")
+                .addToAnnotations("strimzi.io/use-connector-resources", "true")
+            .endMetadata()
+                .editSpec()
+                .addToConfig("group.id", connectClusterName)
+                .addToConfig("offset.storage.topic", connectClusterName + "-offsets")
+                .addToConfig("config.storage.topic", connectClusterName + "-config")
+                .addToConfig("status.storage.topic", connectClusterName + "-status")
+            .endSpec().done();
```

Comment:
Right now both this and the connect test are in following order:
* Create ConnectS2I
* Create Connector
* Create conflicting Connect with the same name.

I wonder if the better order would be:
* Create ConnectS2I
* Create conflicting Connect with the same name.
* Create Connector

Or mybe at least try that for example connector update still works after you create the conflicting Connect.

Reply:
That make sense, I will update it.

Reply:
You probably need to test both to ensure the first-created-resource-wins strategy.